core/trino-main/src/main/java/io/trino/cost/ComparisonStatsCalculator.java

```diff
@@ -240,7 +240,7 @@ private static PlanNodeStatsEstimate estimateExpressionNotEqualToExpression(
         return result.build();
     }
 
-    private static double averageExcludingNaNs(double first, double second)
```

Comment:
Move above private methods.
nit: Consider moving this methods to some `MathUtils` or `StatsUtils` along with `maxExcludeNaN`m `minExcludeNaN`

Reply:
moved to MoreMath